docs/quickstart-docker.md

```diff
@@ -0,0 +1,192 @@
+---
+description: A quick start guide for Featureform with Docker.
+---
+
+# Quickstart (Docker)
+This quickstart uses the standalone containered version of Featureform. It can be used to connect to the same providers 
+as the Kubernetes Hosted version, but lacks the scaling capabilities.
+
+This quickstart will walk through creating a few simple features, labels, and a training from a fraud
+detection dataset using Postgres and Redis.
+
+### Requirements
+
+- Python 3.7+
+- Docker
+
+## Step 1: Install The Featureform CLI
+```shell
+pip install featureform
+```
+
+## Step 2: Pull Containers
+We'll use postgres and redis as providers in this quickstart. We've created a Featureform Postgres container that
+has preloaded data. You can checkout the source data [here](https://featureform-demo-files.s3.amazonaws.com/transactions.csv), 
+we'll just be using a sample of it.
+
+We'll also run the Featureform container in the foreground so logs are available.
+
+```shell
+docker run -d -p 5432:5432 featureformcom/postgres
+docker run -d -p 6379:6379 redis
+docker run -p 7878:7878 -p 80:80 featureformcom/featureform
+```
+
+Note: If using an M1 Mac, run the following command instead for compatibility 
+```shell
+docker run -p 7878:7878 -p 80:80 -e ETCD_ARCH="ETCD_UNSUPPORTED_ARCH=arm64" featureformcom/featureform
+```
+
+Wait 30-60 seconds to allow the Postgres container to fully initialize the sample data. 
```

Comment:
should there be a check here to run the get or list to check that the data is ready

Reply:
This is postgres itself that needs to initialize. Kind of annoying but no great way to check at the moment